warehouse-management-service/main.go

```diff
@@ -4,45 +4,57 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"warehouse-management-service/internal/config"
+	"warehouse-management-service/internal/handler"
+	"warehouse-management-service/pkg/database/postgres"
 
-	"github.com/gin-gonic/gin"
+	_ "github.com/lib/pq"
+)
+
+const (
+	CONFIG_FILE_PATH_ENV_VARIABLE = "CONFIG_FILE_PATH"
 )
 
 func main() {
-	err := initialize()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "App startup error: %v", err)
-		os.Exit(1)
+	// Get absolute path of config file from env variable
```

Comment:
minor: This comment does not add much value in my eyes, the code below it is self-explanatory.

The useful part of the comment (ie, the path being absolute) could be inserted as a godoc comment for the const that is being looked up.

Reply:
Same comment for the other comments in this function. The code's intent is clear at a glance already.

Reply:
Removed these comments